### PR TITLE
DlgReplace

### PR DESCRIPTION
**Pull request by lilyclements**
In Progress
EDIT: This is now ready for review

### CODE REVIEW

File: instat/dlgReplace.vb [hunk 1]
```diff
@@ -158,77 +147,27 @@ Public Class dlgReplace
...
-
-            If (strVarType = "numeric" OrElse strVarType = "integer") Then
-                ucrBase.clsRsyntax.RemoveParameter("new_value")
-                ucrBase.clsRsyntax.AddParameter("new_value", ucrInputNewValue.GetText())
-            Else
-                ucrBase.clsRsyntax.RemoveParameter("new_value")
-                ucrBase.clsRsyntax.AddParameter("new_value", Chr(34) & ucrInputNewValue.GetText & Chr(34))
-            End If
-        End If
+    Private Sub InputValue()
```
Comment (stevekogo) [inline instat/dlgReplace.vb]: This is empty yet you call it in some parts of the code.

File: instat/dlgReplace.vb [hunk 2]
```diff
@@ -253,10 +192,17 @@ Public Class dlgReplace
...
 
-    Private Sub ucrReceiverReplace_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrReceiverReplace.ControlContentsChanged, ucrSelectorReplace.ControlContentsChanged, ucrInputOldValue.ControlValueChanged, ucrInputNewValue.ControlValueChanged, ucrInputRangeFrom.ControlValueChanged, ucrInputRangeTo.ControlValueChanged
-        EnableRange()
-        InputOldValue()
-        InputNewValue()
+    Private Sub ucrReceiverReplace_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReceiverReplace.ControlContentsChanged
         TestOKEnabled()
     End Sub
+
+    Private Sub ucrPnlOld_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlOld.ControlValueChanged, ucrPnlNew.ControlValueChanged
```
Comment (stevekogo) [inline instat/dlgReplace.vb]: you can call ucrReceiverReplace.controlvaluechanged here since it runs similar routines

**Comment by dannyparsons** [comment]
@stevekogo Good to see you are keen! But I don't think this is ready for review yet.

**Comment by stevekogo** [comment]
@dannyparsons Good point, let me reserve my comments till the pull request is open for review.

**Comment by lilyclements** [comment]
@africanmathsinitiative/developers This is ready for review

It says nine files are changed - no need to review the changes to dlgExtremes. This was fixing a conflict

**Comment by lilyclements** [comment]
@africanmathsinitiative/developers This is still ready for review

**Comment by lilyclements** [comment]
@africanmathsinitiative/developers This needs to be merged today for the next version. in the current master copy this is not in a stable state. Can someone please review this ASAP.

File: instat/dlgReplace.vb [hunk 3]
```diff
@@ -15,7 +15,7 @@
 ' along with this program.  If not, see <http://www.gnu.org/licenses/>.
 Imports instat
 Imports instat.Translations
-'Imports RDotNet
+Imports RDotNet
```
Comment (stevenndungu) [inline instat/dlgReplace.vb]: line 17 and 18 could be deleted

**Comment by stevenndungu** [comment]
Apart from that the dialogue will be ready for Danny

**Comment by lilyclements** [comment]
Excellent, good spot. @dannyparsons this is ready to review

**Comment by lilyclements** [comment]
@stevenndungu Can you let Danny know whether this is ready to review, or let me know if I need to do any further changes. Thanks

**Comment by stevenndungu** [comment]
@dannyparsons  this is ready for you to review

**Pull request merged**